great_ape_safe/ape_api/ebtc.py

```diff
@@ -1060,6 +1206,38 @@ def collSurplusPool_sweep_token(
             target.sweepToken(token_address, value)
             assert token.balanceOf(fee_recipient) - balance_before == value
 
+    #### ===== BATCH OPERATIONS ===== ####
+
+    def batch_collateral_feed_source_and_redemption_fee_floor(
+        self, value, new_fee_floor, salt=EmptyBytes32, use_high_sec=False
```

Comment:
perhaps `value` here should have a more implicit meaning of what it really represents as argument later on `setCollateralFeedSource`, no?

Reply:
specially that is a boolean. something reframing more `enable_dynamic_feed` or something along those lines

Reply:
btw, in which cases do we envision that we will batch these sort of actions and which other types of actions is worth having its own batching method that we foreseen happening in one go?

Reply:
could imagine something similar to be batch for `FL_FEE_ADMIN`? perhaps worth having its own batch method as well, wdyt?

Reply:
Good call, changed variable name.

As for other cases, I did think that having the FL fee batched may or may not, I can see the fees for flashloans and flashmint not necessarily have to be the same. We should develop other instances as they become necessary. In this case, it is strictly necessary for these two changes to happen simultaneously. The reason for this is that if the collateral feed is enabled, it will virtually add a "third feed to the mix". Since each of the feeds has a 0.5% Max dev threshold, the new total max dev threshold will be of 1.5%. In such case, the redemption fee floor should be made of the same value in order to protect users from unfair arbitraging and extraction of value: https://docs.ebtc.finance/ebtc/protocol-mechanics/minimized-governance#redemption-fee-floor